README.md

```diff
@@ -140,6 +140,7 @@ Validator                               | Description
 **isMultibyte(str)**                    | check if the string contains one or more multibyte chars.
 **isNumeric(str [, options])**                      | check if the string contains only numbers.<br/><br/>`options` is an object which defaults to `{no_symbols: false}`. If `no_symbols` is true, the validator will reject numeric strings that feature a symbol (e.g. `+`, `-`, or `.`).
 **isOctal(str)**                        | check if the string is a valid octal number.
+**isPassportNumber(str, countryCode)    | check if the string is a valid passport number relative to a specific country code.
```

Comment:
@profnandaa Could you just update `README.md` at master branch and change/fix this line to the correct styling, i.e. 
**isPassportNumber(str, countryCode)**

Reply:
Oops, will do that.